File: test-suite-docs-java/src/test/java/io/micronaut/mcp/docs/tools/jsonschema/output/FenEvaluationResponse.java
package io.micronaut.mcp.docs.tools.jsonschema.output;

//tag::imports[]
import io.micronaut.jsonschema.JsonSchema;
import io.micronaut.serde.annotation.Serdeable;
import jakarta.validation.constraints.NotBlank;
import org.jspecify.annotations.NonNull;

//end::imports[]
//tag::clazz[]
@Serdeable
@JsonSchema
public record FenEvaluationResponse(
    @NonNull @NotBlank String fen,
    @NonNull @NotBlank String evaluation
) {
}
//end::clazz[]

File: test-suite-docs-kotlin/src/test/kotlin/io/micronaut/mcp/docs/tools/jsonschema/output/FenEvaluationResponse.kt
package io.micronaut.mcp.docs.tools.jsonschema.output

//tag::imports[]
import io.micronaut.jsonschema.JsonSchema
import io.micronaut.serde.annotation.Serdeable
import jakarta.validation.constraints.NotBlank

//end::imports[]
//tag::clazz[]
@Serdeable
@JsonSchema
data class FenEvaluationResponse(
    @field:NotBlank val fen: String,
    @field:NotBlank val evaluation: String
)
//end::clazz[]
